odoo/addons/base/tests/test_image.py

```diff
@@ -3,11 +3,10 @@
 
 import base64
 import io
-import binascii
 
 from PIL import Image, ImageDraw, PngImagePlugin
 
-from odoo import tools
+from odoo.tools import image as tools
```

Comment:
Please import the required functions here, and adapt the calling sites.  The diff will be longer, but the code will be cleaner.